### ISSUE TITLE
Asshole Ban Idea: Playtime Bans

### ISSUE THREAD

**Issue by Mastergodown** [HippieStation/HippieStationdeprecated2020#12740]
[Implementation]: # How should the feature work out? What exactly is it supposed to do, in your initial idea? If an item, can it be crafted/found/bought and for what?

Simple, playtime bans would be applied to jobbans, and cannot be applied to server bans.
Its jobban, but it forces you to actually play the time youve been banned for.

[Why]: # How does your feature improve the game anyhow? Why should it be added?

For when people are really assholes, and cringy head of staff players. In example, asshole RD is being cringe and killing people with bombs for shitty reasons, so playtime ban them from RD and captain, and require them to play 2 hours of sci roles.

[Resources]: # Eventual assets such as icons and sounds which you have and want to suggest related to this feature.

Just use the job restriction that requires you to play a certain amount of a job to allow you to play them. Like how TG requires you to play 13 hours of Sci roles to allow you to play RD.
I couldnt imagine this being too hard to code.

# Be sure to ask some maintainer to label this issue as a feature. Either ping them on our discord or PM them on our forum. Thank you for sharing your ideas!
@HippieStation/maintainers

**Comment by MrEpicSax** [comment]
With the current rules / proposed rule changes in mind I don't think this will be necessary, and it sounds like it's going to benefit those who play often but really hamper those who don't have as much spare time. 

It also sounds to me like it will make breaking the rules less punishing. In the situation you described: " _In example, asshole RD is being cringe and killing people with bombs for shitty reasons, so playtime ban them from RD and captain, and require them to play 2 hours of sci roles._ " they'd likely end up catbanned or server banned for a few days depending on severity and notes, or even a permanent job banned anyway. That's assuming if by "shitty reasons" you mean they were non-valids or he was doing crap worthy of being Rule 1'd. And what can an RD do that a regular ol Scientist can't? Or that an Assistant or Janitor can't if they broke into Science or Chemistry? 

And on job restricting by playtime, I don't think we should be limiting our players for it's own sake. We expect them to have a brain and know when they should and should not do something, like picking HoS after never playing security and going whack-a-mole on the crew perma'ing everyone until they bleed to death in their own cells. 13 hours isn't even really enough to bring them up to a level where they could be a decent RD anyway, assuming they had zero experience with science, and they might know science well but just play a different role. Same goes for for the other heads. But hey maybe they can nut it out and learn a bunch in 10 hours or less, everyone has their own pace so we shouldn't set the standard.

I don't think there's any situation where this idea is more fair than a job ban, cat ban or server ban, heck even just sternly warning them if they're quite new, in terms of appropriate punishment.


TL;DR: Fuck NASCAR restrictor plates and tapered spacers are the gayest things to ever happen to the sport just go on youtube and watch this shit instead, way more action: https://www.youtube.com/watch?v=oR-Jud1YZ0A

**Comment by carlopault** [comment]
-2

**Comment by AnthonyJNgo** [comment]
We can just job ban people and have it tick down all the time, only counting bans during active play is a bit excessive punishment imo.

edit: my vote is obviously -3

**Comment by Mastergodown** [comment]
Welp nevermind then
